glide-core/src/request_type.rs

```diff
@@ -223,6 +224,7 @@ impl From<::protobuf::EnumOrUnknown<ProtobufRequestType>> for RequestType {
             ProtobufRequestType::RPushX => RequestType::RPushX,
             ProtobufRequestType::LPushX => RequestType::LPushX,
             ProtobufRequestType::Blpop => RequestType::Blpop,
+            ProtobufRequestType::SInter => RequestType::SInter,
```

Comment:
What is the point of this line. I don't remember seeing you add this line in other PR's if I am not mistaken.

Reply:
This is new in upstream came from #1208